src/robot.coffee

```diff
@@ -408,11 +408,13 @@ class Robot
   #
   # Returns nothing.
   setupExpress: ->
-    user    = process.env.EXPRESS_USER
-    pass    = process.env.EXPRESS_PASSWORD
-    stat    = process.env.EXPRESS_STATIC
-    port    = process.env.EXPRESS_PORT or process.env.PORT or 8080
-    address = process.env.EXPRESS_BIND_ADDRESS or process.env.BIND_ADDRESS or '0.0.0.0'
+    user       = process.env.EXPRESS_USER
+    pass       = process.env.EXPRESS_PASSWORD
+    stat       = process.env.EXPRESS_STATIC
+    port       = process.env.EXPRESS_PORT or process.env.PORT or 8080
+    address    = process.env.EXPRESS_BIND_ADDRESS or process.env.BIND_ADDRESS or '0.0.0.0'
+    limit      = process.env.EXPRESS_LIMIT or '100kb'
+    paramLimit = process.env.EXPRESS_PARAMETER_LIMIT or 1000
```

Comment:
I think you are right. If the env `EXPRESS_PARAMETER_LIMIT` is not a number, `NaN` or `undefined` will return just `NaN` and that `or` 1000 will give us `1000`. Good catch! :+1: